daphne/src/vdaf/mod.rs

```diff
@@ -1685,21 +1674,20 @@ mod test {
         fn unwrap_continued(self) -> (DapAggregationJobState, M) {
             match self {
                 Self::Continued(state, message) => (state, message),
-                _ => panic!("unexpected transition"),
+                Self::Finished(_, _) => panic!("unexpected transition"),
```

Comment:
I generally `..` when I don't want any parameters. But I guess clippy is calling the shots now.

Reply:
actually this was a bit of blind clippy following from me, since this is an unwrap method it should really be a let-else instead of a match